### ISSUE TITLE
Getting following error when trying  to call Azure B2C from angular 2 app.

### ISSUE THREAD

**Issue by djoyrocks** [Srfigie/Angular2-Oauth2-B2C#2]
Hi 
Getting following error when trying  to call Azure B2C from angular 2 app.
http://localhost:55413/#error=unsupported_response_type&error_description=AADB2C90013%253a+The+requested+response+type+'id_token+token'+provided+in+the+request+is+not+supported.%0D%0ACorrelation+ID%253a+260bba8e-2fbc-4056-b565-55ac4c757fec%0D%0ATimestamp%253a+2017-01-04+08%253a55%253a37Z%0D%0A&state=AgF73jh3sDaSAD7fEPzDRgjbokd2FNjkAqaF9a0W

**Comment by Srfigie** [comment]
Hi,
This is an error I was getting too while trying to use a library I found called angular2-oauth2.  I had to modify the library to set the response type to "token id_token" where as the library sets it to "id_token+token" which b2c will reject. (see my modification on line 37 in this repo https://github.com/Srfigie/angular2-oauth2/blob/master/oauth-service.js)
the full repo of the modified packaged can be found here https://github.com/Srfigie/angular2-oauth2

I _thought_ that I had the packages set up in this sample to pull the version of angular2-oauth2 from another git repo I set up instead of the original version. But it would appear your project is using the original library and not the version I modified.  Try the following steps:
1. Uninstall the angular2-oauth2 package from your angular 2 project 
2. Install from the npm console 'npm install git+https://github.com/Srfigie/angular2-oauth2.git"
That should work..
